src/app/components/apparatus-entry/apparatus-entry-detail/apparatus-entry-detail.component.ts

```diff
@@ -0,0 +1,73 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { ApparatusEntry, GenericElement, Reading } from '../../../models/evt-models';
+import { register } from '../../../services/component-register.service';
+import { EVTModelService } from '../../../services/evt-model.service';
+@Component({
+  selector: 'evt-apparatus-entry-detail',
+  templateUrl: './apparatus-entry-detail.component.html',
+  styleUrls: ['./apparatus-entry-detail.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+
+@register(ApparatusEntryDetailComponent)
+export class ApparatusEntryDetailComponent implements OnInit {
+  @Input() data: ApparatusEntry;
+  nestedApps: ApparatusEntry[] = [];
+  rdgHasCounter = false;
+
+  get significantRdg(): Reading[] {
+    return this.data.readings.filter((rdg) => rdg.significant);
+  }
+
+  get notSignificantRdg(): Reading[] {
+    return this.data.readings.filter((rdg) => !rdg.significant);
+  }
+
+  get readings(): Reading[] {
+    return [this.data.lemma, ...this.significantRdg, ...this.notSignificantRdg]
+  }
+
+  get rdgMetadata() {
+    return Object.keys(this.data.attributes).filter((key) => key !== 'id')
+      .reduce((obj, key) => {
+        obj[key] = this.data.attributes[key];
+
+        return obj;
+      },      {});
+  }
```

Comment:
I think it can be changed into 
```
return Object.keys(this.data.attributes).filter((key) => key !== 'id')
  .reduce(
    (obj, key) => ({
      ...obj,
      [key]: this.data.attributes[key],
    }),
    {});
```